docs/modules/ROOT/pages/how-tos/testing/integration/adding.adoc

```diff
@@ -1 +1,46 @@
-= Adding an integration test
\ No newline at end of file
+= Adding an integration test
+
+In {ProductName}, you can add integration tests to verify that the individual components of your application integrate correctly, forming a complete and functional application. {ProductName} runs these integration tests on the container images of components before their deployment.
+
+.Prerequisites
+
+* You have created an application in {ProductName}.
+
+.Procedure
+
+Complete the following steps in the {ProductName} console:
+
+. Open an existing application and go to the *Integration tests* tab.
+
+. Select *Add integration test*.
+
+. In the *Integration test name* field, enter a name of your choosing.
+
+. In the *GitHub URL* field, enter the URL of the GitHub repository that contains the test you want to use.
+. Optional: If you want to use a branch, commit, or version other than the default, specify the branch name, commit SHA, or tag in the *Revisions* field.
+
+. In the *Path in repository* field, enter the path to the `.yaml` file that defines the test you want to use.
+. Optional: To allow the integration tests to fail without impacting the deployment and release process of your application, you can choose to select *Mark as optional for release*.
+
++
+NOTE: By default, all integration test scenarios are mandatory and must pass. A failing integration test marks the application snapshot as failed, preventing its deployment and release. However,  if you have selected *Mark as optional for release*, a failure in this test won't hinder the deployment and release of the application snapshot.
+
+. Select *Add integration test*.
+
+. Start a new build for any component you want to test.
+
+.. For components using the default build pipeline, go to the *Components* tab, select the three dots next to the name of the component, and select *Start new build*.
+
+.. For components with an upgraded build pipeline, make a commit to their GitHub repository.
```

Comment:
We can remove the first part of this sentence. By making a commit, are you suggesting to just push directly to a tracked branch or to make a commit to a source branch for a PR? This is also about trying to trigger the modified configured tests? Can they not be triggered with a comment like `/retest` in a PR or on a commit that has been merged?

Reply:
@dirgim Please let us know your thoughts.

Reply:
Yeah, this is also related to the default build pipelines being deprecated, changing to something along these lines makes sense to me:
```
You can either open a new PR that targets the component's tracked branch or comment `/retest` on an existing PR in order to start the component build and run the new integration test.
```

Reply:
@dirgim This is confusing, sorry!
Do I need to remove line # 11 to line # 34 ?
And then add your suggestion :  You can either open a new PR that targets the component's tracked branch or comment `/retest` on an existing PR in order to start the component build and run the new integration test.

I'm unable to understand the link, because we are talking about steps in then UI and then suddenly we go to commenting in PR.

Reply:
I was thinking of just changing this line to what I proposed, but I can see how suddenly switching to GitHub would be confusing. Maybe something like this:
`The easiest way to start a new component build is to either open a new PR in the GitHub repository that targets the component's tracked branch or comment `/retest` on an existing PR.`